trade-finance-manager-api/src/v1/controllers/utilisation-reports/fee-record-correction/get-fee-record-correction-request-review.controller.test.ts

```diff
@@ -20,20 +20,7 @@ describe('get-fee-record-correction-request-review.controller', () => {
         params: { reportId, feeRecordId, user },
       });
 
-    const aResponseBody = (): FeeRecordCorrectionRequestReviewResponseBody => ({
-      bank: { id: '123', name: 'Test bank' },
-      reportPeriod: {
-        start: { month: 1, year: 2024 },
-        end: { month: 1, year: 2024 },
-      },
-      correctionRequestDetails: {
-        facilityId: '0012345678',
-        exporter: 'A sample exporter',
-        reasons: [RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT],
-        additionalInfo: 'this is the reason',
-        contactEmailAddresses: ['test@test.com'],
-      },
-    });
+    const aResponseBody = (): FeeRecordCorrectionRequestReviewResponseBody => feeRecordCorrectionRequestReviewResponseBodyMock;
```

Comment:
Please so people don't try and modify object used across multiple tests
```suggestion
    const aResponseBody = (): FeeRecordCorrectionRequestReviewResponseBody => { ...feeRecordCorrectionRequestReviewResponseBodyMock };
```

Reply:
done